apps/workbench/config/application.yml.example

```diff
@@ -24,6 +24,9 @@ production:
   secret_token: ~
 
   # You probably also want to point to your API server.
-  arvados_login_base: https://arvados.local:3030/login
-  arvados_v1_base: https://arvados.local:3030/arvados/v1
+  arvados_login_base: https://uuid_prefix.your.domain/login
+  arvados_v1_base: https://uuid_prefix.your.domain/arvados/v1
   arvados_insecure_https: false
+
+  # You might also want to name your workbench
+  site_name: ~
```

Comment:
IMHO, the production section of this file should contain the essential options, as well as the ones which the user is mostly interested in. Also, anything noted in the install documentation should appear here with or without comment. So the user can be sure he is changing the right file and the right options, without fearing he just added another unrecognizable row.